src/main/java/com/uid2/shared/attest/UidCoreClient.java

```diff
@@ -64,11 +65,16 @@
     private final boolean enforceHttps;
     private static boolean useSecureParameters = true;
     private boolean allowContentFromLocalFileSystem = false;
+    private AtomicReference<Handler<Integer>> responseWatcher;
 
     public static UidCoreClient createNoAttest(String attestationEndpoint, String userToken, ApplicationVersion appVersion, boolean enforceHttps) {
         return new UidCoreClient(attestationEndpoint, userToken, appVersion, CloudUtils.defaultProxy, new NoAttestationProvider(), enforceHttps);
     }
 
+    public static UidCoreClient createFakeForTest() {
```

Comment:
Is there a way to avoid putting unit test logic into prod code? Can this be solved by a mock?

Reply:
moved to test code.

Reply:
moved to test code.